desktop_version/src/Font.cpp

```diff
@@ -765,13 +765,13 @@ static Font* container_get(FontContainer* container, uint8_t idx)
     return NULL;
 }
 
-static Font* fontsel_to_font(int sel, bool* rtl)
+static Font* fontsel_to_font(int sel, bool* rtl, bool custom)
 {
     /* Take font selection integer (0-31) and turn it into the correct Font
      * 0: PR_FONT_INTERFACE - use interface font
      * 1: PR_FONT_LEVEL     - use level font
      * 2: PR_FONT_8X8       - use 8x8 font no matter what
-     * 3-31:                - use (main) font index 0-28
+     * 3-31:                - use font index 0-28
```

Comment:
I would update this comment with some more explanation what "font index" now means, namely that it's affected by the `custom` argument (which in turn depends on the newly added flag)